src/redux/reducers/basket/actionTypes.tsx

```diff
@@ -0,0 +1,5 @@
+export const ADD_JEWELRY_TO_BASKET = 'ADD_JEWELRY_TO_BASKET';
```

Comment:
Here an example of how to create typed actions:
```
const typesNames = [
  'LOGIN',
  'LOGIN_SUCCESS',
  'LOGIN_FAILURE',
  'LOGOUT',
  'CHECK_AUTH',
  'SET_AUTH',
  'GET_CURRENT_USER_SUCCESS',
  'GET_CURRENT_USER_FAILED',
  'DELETE_ERROR_MESSAGE',
] as const;

export type AuthActionType = {
  [Key in typeof typesNames[number]]: string;
};

export const prefix: string = 'AUTH';

export const authActionTypes: AuthActionType = createConstants<AuthActionType>(
  typesNames,
  prefix,
);

```
createConstants:
```
export const createConstants = <T>(
  constants: readonly any[],
  constantType: string,
): T =>
  constants.reduce((acc, constant: string) => {
    const prefix = constantType ? `${constantType}/` : '';
    acc[constant] = `${prefix}${constant}`;
    return acc;
  }, {});
```

All actions will be available inside authActionTypes object.